src/store/reducers/devicesReducer.js

```diff
@@ -82,6 +82,22 @@ const insertDeviceReducer = (state, { payload }) => {
   });
 };
 
+const updateTagsReducer = (state, { payload }) => {
+  let updatedTagData = {};
```

Comment:
const

Reply:
Except, again... it gets updated. What am I missing here?

Reply:
See answer above